modules/oadp-release-notes-1-1-2.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-1-2_{context}"]
+= OADP 1.1.2 release notes
+
+The OADP 1.1.2 release notes include product recommendations, a list of fixed bugs and descriptions of known issues.
+
+== Product recommendations
+
+To prepare for the upgrade from VolSync 0.5.1 to 0.6.0, in the `openshift-adp` namespace you must add this annotation.
```

Comment:
```suggestion
To prepare for the upgrade from VolSync 0.5.1 to 0.6.0, you must add this annotation in the `openshift-adp` namespace by running the following command:
```

Reply:
change to the latest available volsync from the volsync stable channel.

Reply:
@weshayutin 
Like this? 
"...the upgrade from VolSync 0.5.1 to the latest available volSync from the VolSync stable channel"

Reply:
ya.. that's fine..  I want to make sure we're not suggesting 0.6.0 is the required version.  0.6.1 is going to be published in a few days.

Reply:
Done